api/AmbientLightSensor.json

```diff
@@ -0,0 +1,130 @@
+{
+  "api": {
+    "AmbientLightSensor": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/AmbientLightSensor",
+        "support": {
+          "webview_android": {
+            "version_added": "54"
+          },
+          "chrome": {
+            "version_added": "54"
+          },
+          "chrome_android": {
+            "version_added": "54"
+          },
+          "firefox": {
+            "version_added": null
+          },
+          "firefox_android": {
+            "version_added": null
+          },
+          "ie": {
+            "version_added": null
+          },
+          "opera": {
+            "version_added": null
+          },
+          "opera_android": {
+            "version_added": null
+          },
+          "safari": {
+            "version_added": null
+          },
+          "safari_ios": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "AmbientLightSensor": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/AmbientLightSensor/AmbientLightSensor",
+          "support": {
+            "webview_android": {
+              "version_added": "54"
+            },
+            "chrome": {
+              "version_added": "54"
+            },
+            "chrome_android": {
+              "version_added": "54"
+            },
+            "firefox": {
+              "version_added": null
+            },
+            "firefox_android": {
+              "version_added": null
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": null
+            },
+            "opera_android": {
+              "version_added": null
+            },
+            "safari": {
+              "version_added": null
+            },
+            "safari_ios": {
+              "version_added": null
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": false,
```

Comment:
as far as I can tell from [the spec](https://developer.mozilla.org/en-US/docs/Web/API/AmbientLightSensor/AmbientLightSensor#Specifications), this _is_ a standardized constructor.